src/elasticsearch/addl_index_transformations/portal/add_partonomy.py

```diff
@@ -13,23 +37,31 @@ def add_partonomy(doc):
     ... }
     >>> from json import dumps
     >>> doc = {
+    ...     'origin_sample': {
+    ...         'organ': 'LI'
+    ...     },
     ...     'rui_location': dumps(rui_location)
     ... }
     >>> add_partonomy(doc)
     >>> del doc['rui_location']
+    >>> del doc['origin_sample']
     >>> doc
     {'anatomy_0': 'body', 'anatomy_1': 'large intestine', 'anatomy_2': 'transverse colon'}
```

Comment:
These are two separate execution paths, so we need two separate tests, at least.